packages/zowe-explorer/__tests__/__unit__/utils/ProfileUtils.unit.test.ts

```diff
@@ -75,4 +78,14 @@ describe("ProfileUtils.writeOverridesFile Unit Tests", () => {
         spy.mockClear();
         spyRead.mockClear();
     });
+    it("should log error details", async () => {
+        createBlockMocks();
+        const errorDetails = new Error("i haz error");
+        const label = "test";
+        const moreInfo = "Task failed successfully";
+        await errorHandling(errorDetails, label, moreInfo);
+        expect(globals.LOG.error).toBeCalledWith(
+            `Error: ${errorDetails.message}\n` + JSON.stringify({ errorDetails, label, moreInfo })
```

Comment:
see other comment

Reply:
Perhaps I should change the other place to reference `${errorDetails.message}` to make it more obvious what it's doing, and more consistent with this.